### PR TITLE
Animals Table

### PR DESCRIPTION
**Pull request by dheerajsachdeva**
- Create table animals 
- insert data into animals 
- implement the required queries 

Screenshot for all my queries:

image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by TuyishimireEric** [comment]
# INVALID ❌ ❌ ❌
### Hi @dheerajsachdeva  👋

While you made a great effort in this project, I can't go ahead and review your code.

## Invalid Code Review Request ❌
You have submitted a project that doesn't have a professional README 😨, Kindly use the Microverse README template with personalized content and submit again once you are done 🙏  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Kindly create  a new PR that follows the Gitflow approach

Cheers, and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

---
_Invalid Code Review Request does not count into the code reviews limit._

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
+create database vet_clinic
+
+create table animals (
+id int PRIMARY KEY,
+name varchar(50),
+date_of_birth date,
+escape_attempts int,
+neutered boolean,
+weight_kg decimal
+);
```
Comment (pinkmoon25) [inline schema.sql:12]: - [ ] [OPTIONAL] good job :+1: , Just one suggestion you can make the ID column NOT NULL and AUTO_INCREMENT: this means that the id value will be generated automatically by the database system for each new row added to the table.

**Review by pinkmoon25 — APPROVED** [review]
Hi @dheerajsachdeva , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Gitflow :heavy_check_mark: 
Data :heavy_check_mark: 
Schema and query :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (dheerajsachdeva) [reply schema.sql:12]: @PinkMoon25 Ok, Will do That. Thanks. 😄

**Pull request merged**